tests/integration/helpers.bash

```diff
@@ -449,17 +449,15 @@ function setup_debian() {
 
 	setup_recvtty
 	run mkdir "$DEBIAN_BUNDLE"
-
-	if [ ! -d "$DEBIAN_ROOTFS/rootfs" ]; then
-		get_and_extract_debian "$DEBIAN_BUNDLE"
+	run mkdir "$DEBIAN_BUNDLE"/rootfs
+	if [ ! -e "$DEBIAN_IMAGE" ]; then
+		echo "debian image does not exist"
+		exit 3
 	fi
-
-	# Use the cached version
-	if [ ! -d "$DEBIAN_BUNDLE/rootfs" ]; then
-		cp -r "$DEBIAN_ROOTFS"/* "$DEBIAN_BUNDLE/"
-	fi
-
+	tar --exclude './dev/*' -C "$DEBIAN_BUNDLE"/rootfs -xf "$DEBIAN_IMAGE"
 	cd "$DEBIAN_BUNDLE"
+	runc_spec
+	update_config '.root.readonly |= false'
```

Comment:
Not sure where this is coming from, can you explain?

Reply:
I fix a test error:
```
not ok 44 runc run (hooks library tests)
2994# (in test file tests/integration/hooks.bats, line 54)
2995# `[ "$status" -eq 0 ]' failed
2996# runc list (status=0):
2997# ID PID STATUS BUNDLE CREATED OWNER
2998# runc spec (status=0):
2999# 
3000# /tmp/bats.8004.src: line 47: warning: here-document at line 43 delimited by end-of-file (wanted `EOF')
3001# runc run test_debian (status=1):
3002# Running hook #0:: error running hook: exit status 1, stdout: , stderr: ldconfig: Can't link /lib/librunc-hooks-create-runtime.so.1 to librunc-hooks-create-runtime.so.1.0.0
3003# ldconfig: Can't link /lib/librunc-hooks-create-container.so.1 to librunc-hooks-create-container.so.1.0.0
3004# ldconfig: Can't create temporary cache file /etc/ld.so.cache~: Read-only file system
3005# runc list (status=0):
3006# ID PID STATUS BUNDLE CREATED OWNER
3007ok 45 kill detached busybox
```

Reply:
This should be done in the very test that fails. You have done it globally, not good.

Reply:
OK, I will fix it.